doc/api/repl.md

```diff
@@ -283,7 +306,20 @@ applications.
 The following illustrates a hypothetical example of a REPL that performs
 translation of text from one language to another:
 
-```js
+```mjs
+import repl from 'node:repl';
+import { Translator } from 'translator';
+
+const myTranslator = new Translator('en', 'fr');
+
+function myEval(cmd, context, filename, callback) {
+  callback(null, myTranslator.translate(cmd));
+}
+
+repl.start({ prompt: '> ', eval: myEval });
+```
```

Comment:
this code block adds a mock dependency to a module that _actually_ exists and is very dormant ([`translator`](https://www.npmjs.com/package/translator)). IMO we should try to avoid that _and_ the alternative of using a module that doesn't exist.

While this is also true of the previously existing example, since it's being updated now with a totally new example I think it's a good time to adjust it.

Reply:
Agreed! I also thought about that example since it throws an error when you try to execute it as is and new users can be frustrated by that, I'll come up with a good alternative and update as soon as I can! Thanks for the feedback! :raised_hands:

Reply:
I've updated the example, tell me what you think! :blush: And again, thank you for taking the time to provide feedback! :pray: